js/components/directionControl.js

```diff
@@ -0,0 +1,39 @@
+var React = require('react');
+var classnames = require('classnames');
+
+var DirectionControl = React.createClass({
```

Comment:
Can we change this to DirectionControlVr or something similar? To be clear that it is a VR related functionality

Reply:
\+